Sources/SwiftProtobuf/Google_Protobuf_FieldMask+Extensions.swift

```diff
@@ -137,7 +137,7 @@ public extension Google_Protobuf_FieldMask {
   ///   defined using the JSON names for the fields.
   public init?(jsonPaths: String...) {
     // TODO: This should fail if any of the conversions from JSON fails
-    self.init(protoPaths: jsonPaths.flatMap(JSONToProto))
+    self.init(protoPaths: jsonPaths.compactMap(JSONToProto))
```

Comment:
Does this work on older Swift versions or does this need to be conditional?

Reply:
It does not, see PR summary.